packages/ui/src/theme/overwrites/components/link.ts

```diff
@@ -0,0 +1,7 @@
+export const Link = {
+  baseStyle: {
+    _hover: {
+      textDecoration: 'none',
```

Comment:
💭 In many cases we probably want links to have an underline. It could be better to remove this Link-related styling directly in the TeamCard component.